commands/issue.go

```diff
@@ -101,10 +107,18 @@ With no arguments, show a list of open issues.
 		Open the new issue in a web browser.
 
 	-M, --milestone <ID>
-		Add this pull request to a GitHub milestone with id <ID>.
+		Display only issues for a GitHub milestone with id <ID>.
+
+		When opening an issue, add this issue to a GitHub milestone with id <ID>.
 
 	-l, --labels <LABELS>
-		Add a comma-separated list of labels to this issue.
+		Display only issues with certain labels.
+
+		When opening an issue, add a comma-separated list of labels to this issue.
+
+	-t, --since <TIME>
+		Display only issues updated at or after a certain time. The time is a
+		timestamp in ISO-8601 format: YYYY-MM-DDTHH:MM:SSZ.
```

Comment:
The GitHub API already supports the YYYY-MM-DD format, so I've updated the doc, but no need to write more code.